docs/source/usage/usage.rst

```diff
@@ -54,7 +54,7 @@ The address for a Vendor should be the actual location where they sell their Pro
 
 **Pre-requisites**
 
-Vendors sell Products, so in order to create a new Vendor, some Products must exist (ideally, the specific Products that Vendor sells). Before creating a new Vendor, it is a good idea to make sure their Products exist. The Vendor form *requires* at least one Product to be added to the Vendor. A Vendor's Product list can be changed later.
+Vendors sell Products, so in order to create a new Vendor, some Products must exist (ideally, the specific Products that Vendor sells). Before creating a new Vendor, it is a good idea to make sure their Products exist if not you can do so by creating one directly from the vendor's from itself. The Vendor form *requires* at least one Product to be added to the Vendor. A Vendor's Product list can be changed later.
```

Comment:
I would scrap most of this sentence. Mostly it's important that the Vendor form requires at least one product, now.